cmd/utils.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// ProjectType ...
+type ProjectType string
+
+const (
+	// iOS
+	xcodeWorkspace ProjectType = "*.xcworkspace"
+	xcodeProject   ProjectType = "*.xcodeproj"
+)
+
+// Scans the root dir for the provided project files
+// If none of them in the root dir, then it will return an error
+func scanForProjectFiles(projectFiles []ProjectType) (string, error) {
```

Comment:
i would recommend using this project file search logic: https://github.com/bitrise-core/bitrise-init/blob/master/scanners/xamarin/xamarin.go#L88-L100
we use this for a long time ago it also takes care of some edge cases (solution is filtered in node modules and components directory)